terraform/application/database.tf

```diff
@@ -0,0 +1,32 @@
+module "postgres" {
+  source = "./vendor/modules/aks//aks/postgres"
+
+  namespace                   = var.namespace
+  environment                 = var.environment
+  azure_resource_prefix       = var.azure_resource_prefix
+  service_name                = var.service_name
+  service_short               = var.service_short
+  config_short                = var.config_short
+  cluster_configuration_map   = module.cluster_data.configuration_map
+  use_azure                   = var.deploy_azure_backing_services
+  azure_enable_monitoring     = var.enable_monitoring
+  azure_enable_backup_storage = var.enable_postgres_backup_storage
+  server_version              = "16"
+}
+
+
+module "redis-cache" {
```

Comment:
redis not required I believe, so can be removed

Reply:
I don't think we need or should be heading towards redis, if we do then we add it then.

cc @tomtrentham